packages/ui/src/providers/api/context.tsx

```diff
@@ -3,6 +3,5 @@ import { UseApi } from './provider'
 
 export const ApiContext = createContext<UseApi>({
   isConnected: false,
-  state: 'CONNECTING',
```

Comment:
Can't we detect connecting state with RxApi?

Reply:
Probably everything between starting the connection and `isConnected` observable emits change is a connecting state. For now, it was not used so I've removed the `state` variable.

Reply:
I'd go with a holistic approach to connection status improvements: https://github.com/Joystream/pioneer/issues/76. For now, I'm OK with a current connection state handling.

Reply:
OK